end-to-end/README.md

```diff
@@ -4,7 +4,8 @@ All the instructions on this page refer to the Docker-based deployment. To use
 the native deployment, just substitute "docker" with "native" in all the
 commands given below.  For example, `make native-deploy` instead of `make
 docker-deploy`, and `./end-to-end-native provision` instead of `./end-to-end-docker
-provision`
+provision`. The same applies to `./end-to-end-aws` with the proviso that
```

Comment:
Did you mean the same applies to `aws` meaning replace in all scripts with aws instead of using docker? It is not super clear whether the guidance is specific to ./end-to-end command or it follows from the previous text. Could you please clarify the wording?

Reply:
Updated.